controller.java

```diff
@@ -2,13 +2,17 @@
 public class controller {
     public static void main(String[] args) {
         // Declare two numbers
-        int num1 = 10;
-        int num2 = 20;
+        int num1 = 40;
+        int num2 = 40;
```

Comment:
change variable name to number2
```suggestion
        int number2 = 40;
```